nav_msgs/OccupancyGrid map
rbe_3002_final/keypoint[] keypoints
